idd/Energy+.idd.in

```diff
@@ -20878,15 +20884,45 @@ People,
        \key KSU
        \key AdaptiveASH55
        \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
        \note optional (fourth thermal comfort model and report type)
-  A18; \field Thermal Comfort Model 5 Type
+  A18, \field Thermal Comfort Model 5 Type
        \type choice
        \key Fanger
        \key Pierce
        \key KSU
        \key AdaptiveASH55
        \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
        \note optional (fifth thermal comfort model and report type)
+  A19, \field Thermal Comfort Model 6 Type
+       \type choice
+       \key Fanger
+       \key Pierce
+       \key KSU
+       \key AdaptiveASH55
+       \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
+       \note optional (fifth thermal comfort model and report type)
```

Comment:
Either delete this note or change fifth --> sixth
Similar for the next field.

Reply:
Fixed.